documentation/navitia.io/source/integration.rst

```diff
@@ -355,6 +356,21 @@ Warning: this last request gives the stop areas used by the vehicle
 journeys containing the headsign `CANE`, *not* the stop areas where it
 exists a stop time with the headsign `CANE`.
 
+since / until
+_____________
+
+- Type: `datetime`
+
+To be used only on a vehicle journey collection, to filter on a period.
+Both parameters until and since are optional.
```

Comment:
Enhance parameter please. For example:
_Both parameters "until" and "since" are optional._